utils/run_covent_tests.py

```diff
@@ -43,9 +43,13 @@ def run(args):
     else:
         run("set gui_fio2 21")
 
-    # Ensure the ventilator is blowing with some params.  Edwin's request is
-    # that the ventilator is continuously on during these tests.
-    run("set gui_mode 1")
+    # Ensure the ventilator fan is on, and disconnect the fan from the system
+    # so it's not inflating the test lung.  Edwin's request is that the fan
+    # doesn't have to spin up during these tests.
+    run("set forced_exhale_valve_pos 1")
+    run("set forced_blower_valve_pos 0")
+    run("set forced_blower_power 1")
```

Comment:
Will these forced values be unforced by the code inside the preset so that we're testing normal ventilator operation and not the forced values?

Reply:
That's what this does

```
    run("set forced_exhale_valve_pos -1")
    run("set forced_blower_valve_pos -1")
    run("set gui_mode 1")
    run("set forced_blower_power -1")
```

Guess it's not obvious.  Added a comment.